apps/antalmanac/src/lib/termData.ts

```diff
@@ -29,6 +29,7 @@ class Term {
  * Months are 0-indexed
  */
 const termData = [
+    new Term('2025 Winter', '2025 Winter Quarter', [2025, 0, 6], [2025, 2, 21]),
```

Comment:
Not sure what the finals date is used for, but comment on L28 mentions it should be the first day (sat). This would correspond to Mar 15 (`[2025, 2, 15]`).

Reply:
Feel free to dismiss my review to merge this PR if it's not important.